server/src/server.js

```diff
@@ -23,6 +23,14 @@ const jspath = "/js/",
   footer = "</body></html>",
   script = `
 function gotData(Handler, data) {
+  data.linkify = function (url) {
+    if (url.indexOf("?") !== -1) {
+      url += "&gitRevision=" + gitRevision;
+    } else {
+      url += "?gitRevision=" + gitRevision;
+    }
+    return url;
+  }
```

Comment:
The function needs to be defined both in the client and the server rendering paths. I'll see if I can move it somewhere that it is not duplicated.